src/main/java/pl/jkap/sozzt/contract/domain/ContractMapper.java

```diff
@@ -0,0 +1,53 @@
+package pl.jkap.sozzt.contract.domain;
+
+import lombok.AllArgsConstructor;
+import org.hibernate.cfg.NotYetImplementedException;
+import pl.jkap.sozzt.contract.dto.ContractDto;
+import pl.jkap.sozzt.contract.dto.ContractStepEnum;
+
+import static java.util.Objects.requireNonNull;
+
+@AllArgsConstructor
+class ContractMapper {
```

Comment:
any reason why not use mapstruct?

Reply:
I wrote about it there: https://github.com/szpalajoan/sozzt/pull/1/files#r961324501